Package catala.0.6.0

denismerigoux opened the pull request:
### `catala.0.6.0`
Compiler and library for the literate programming language for tax code specification
Catala is a domain-specific language for deriving faithful-by-construction algorithms from legislative texts. See https://catala-lang.org for more information



---
* Homepage: https://github.com/CatalaLang/catala
* Source repo: git+https://github.com/CatalaLang/catala.git
* Bug tracker: https://github.com/CatalaLang/catala/issues

---
:camel: Pull-request generated by opam-publish v2.0.3

kit-ty-kate commented:
Your test depends on a non existing package. Should `ninja_utils` really be in opam-repository or should it stay in this repository? If it’s the later, you might want to remove the `ninja_utils.opam` file and remove the `(public_name ninja_utils)` from your `build_systems/dune` file.
```
#=== ERROR while compiling catala.0.6.0 =======================================#
# context              2.0.10 | linux/x86_64 | ocaml-base-compiler.4.11.2 | pinned(https://github.com/CatalaLang/catala/archive/0.6.0.tar.gz)
# path                 ~/.opam/4.11/.opam-switch/build/catala.0.6.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p catala -j 31 @install @runtest
# exit-code            1
# env-file             ~/.opam/log/catala-1751-5f1023.env
# output-file          ~/.opam/log/catala-1751-5f1023.out
### output ###
# File "build_system/tests/dune", line 3, characters 34-45:
# 3 |  (libraries alcotest clerk.driver ninja_utils catala.utils))
#                                       ^^^^^^^^^^^
# Error: Library "ninja_utils" not found.
# -> required by _build/default/build_system/tests/test_clerk_driver.exe
# -> required by alias build_system/tests/runtest in build_system/tests/dune:2
# (cd _build/default/compiler/surface && /home/opam/.opam/4.11/bin/menhir tokens.mly parser.mly --base parser --compile-errors parser.messages) > _build/default/compiler/surface/parser_errors.ml
# Read 138 sample input sentences and 138 error messages.

'opam install -v --with-test catala.0.6.0' failed.
```

denismerigoux commented:
Hi @kit-ty-kate, thanks for your review and comment ! I identified this error this morning and fixed it https://github.com/CatalaLang/catala/commit/c5d63148f8c70b6c590bf66618cfcf93dd425e18, and also updated the release (second commit on this PR). The new CI run looks pretty good to me : 

https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/44ea7c2b2981b399a12862068042e86f44b84e70

kit-ty-kate commented:
Thanks! Could you return the fixes to the opam file so they can be taken into account for next time?

denismerigoux commented:
Yep will do!
